src/layouts/HeroTwoColLayout.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <QPage class="stl-footer--bottom">
+    <section class="hero hero__background-container">
+      <div class="q-pa-md">
+        <div class="row items-center">
+          <div class="col-md-6">
+            <q-img
+              src="statics/images/banner-img.jpg"
+            />
+          </div>
+          <div class="col-md-6">
+            <div class="hero__background bg-primary-gradient" />
+            <div class="hero__content text-center text-white">
```

Comment:
I suggest you reuse existing `BasicHeroLayout.vue` since there is a slot to easily customize content.

https://github.com/abruere/workingoo/blob/e89998f6ed2ecfd52237da702b2eb2d25d304fe3/src/layouts/BasicHeroLayout.vue#L3-L8

You can add an outer slot in `BasicHeroLayout.vue` if you need more customization

```vue
<section class="hero hero__background-container"> 
   <div class="hero__background bg-primary-gradient" /> 
   <slot name="heroContentContainer">
     <!-- This is default content of heroContentContainer slot -->
     <div class="hero__content text-center text-white"> 
       <slot name="heroContent" /> 
     </div> 
  </slot>
</section> 
```

Note: you can only use _either_ `heroContent` or `heroContentContainer`, it’s all what you need to fully customize the content and keep current hero background.

Reply:
I worked from your changes and reuse 'BasicHeroLayout.vue'. Based on 'this.route' element is displayed.